db/re/item_db_usable.yml

```
@@ -78855,6 +78855,13 @@ Body:
       NoAuction: true
     Script: |
       getgroupitem(IG_STOVE_CHANNELING_BOX_4);
+  - Id: 103263
+    AegisName: Summer_Select_Box4
+    Name: Star Footprint Selection Box (Summer Beach)
+    Type:  Usable
+    Weight: 1
+#   Script: |
+#     /* TODO */
```

```suggestion
    Weight: 10
```
The item selection box does not require a call from a script command. The item will open the UI using the items defined in `item_packages.yml` and by the client